source/developers/how-tos/get-ready-for-python-dev.rst

```diff
@@ -0,0 +1,82 @@
+How To Get Ready for Python Development
+#######################################
+
+.. How-tos should have a short introduction sentence that captures the user's goal and introduces the steps.
+
+This how-to will help prepare your local system to start developing on a Python-based service that is part of the Open edX Platform.
+
+Assumptions
+***********
+
+.. This section should contain a bulleted list of assumptions you have of the
+   person who is following the How-to.  The assumptions may link to other
+   how-tos if possible.
+
+* You have Git installed
+* You have Python 3.8 installed
+
+Steps
+*****
+
+.. note::
+
+   Some repositories will have a variation of these instructions in their README. Often it will be better to follow the instructions here instead, as the README instructions may be outdated, but keep an eye out for repositories that document special workflows.
```

Comment:
I get what you're saying here, but from the perspective of a new Python dev who has no idea what special workflows are  (which IMO is the audience of this doc), this note amounts to saying:

> If these instructions conflict with the repository's instructions, then follow these instructions, except in the cases where you shouldn't, and in those cases you should follow repository's instructions.

which I think is more confusing and disheartening than it is helpful 😛

Reply:
It's tough to figure out how to say "a lot of those READMEs are wrong but some of them are not" in a useful way and that, frankly, isn't too embarrassing. Maybe this:

> These instructions will not work in all repositories. If they do not, check the repo's README for a Getting Started section for possible alternative instructions.

Reply:
I actually really like this wording!